### ISSUE TITLE
Operator parameter constraint link

### ISSUE THREAD

**Issue by jsntsay** [IBM/lale#722]
For each operator, if a parameter has a constraint (such as penalty and loss in [LinearSVC](https://lale.readthedocs.io/en/latest/modules/lale.lib.sklearn.linear_svc.html?highlight=linearsvc)), the parameter description should have some indicator that the constraint exists and/or a link to the list of constraints.

**Comment by hirzel** [comment]
For some reason, the embedded links in [PCA](https://lale.readthedocs.io/en/latest/modules/lale.lib.sklearn.pca.html) seem to point to SnapLogisticRegression. Need to investigate.

**Comment by hirzel** [comment]
Fixed with PR #761
